runtime/src/lib.rs

```diff
@@ -518,8 +519,11 @@ impl pallet_domains::Config for Runtime {
 	type DomainsInsertLimit = DomainsInsertLimit;
 	type RegistrationPeriodLimit = RegistrationPeriodLimit;
 	type MaxOuterValueLength = MaxOuterValueLength;
+	type MaxRecordKeySize = ConstU32<128>;
+	type MaxRecordValueSize = ConstU32<4096>;
```

Comment:
Why 4k bytes? BTW, are these values inspired by ENS or something?

Reply:
I think in ENS it's byte32

Reply:
it's just a number that we can increase

Reply:
What values you recommend ?